src/views/tickets/JSViewer.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+
+import { Dropdown } from 'semantic-ui-react';
```

Comment:
Is there a reason for these being spaced out?

Reply:
I don't have any JS programming experience before, so I followed the existing coding style.

Reply:
No worries - take the comments I’m leaving as optional. I’m just pointing out things I see that look odd as I scan through it - these aren’t criticisms, it’s very good Javascript for you being new to the language.